actionpack/lib/action_controller/metal/url_for.rb

```diff
@@ -20,6 +20,24 @@ module ActionController
   #       @url        = root_path # named route from the application.
   #     end
   #   end
+  #
+  # This module also defines the url option +_recall with the path parameters
```

Comment:
We should not talk about `_recall` here, it is internal and should not appear in the public documentation.